client/src/App.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import axios from 'axios';
+import SearchBox from './components/SearchBox.jsx';
+import BreakdownBox from './components/BreakdownBox.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+      super(props)
+      this.state = {
+          allReviews: [],
+          avgRating: 0,
+          search: ''
+      }
+      this.getReviews = this.getReviews.bind(this);
+      this.handleClick = this.handleClick.bind(this);
+      this.findAvgReviewRating = this.findAvgReviewRating.bind(this);
+      this.countRecommendedReviews = this.countRecommendedReviews.bind(this);
+      this.handleSearchChange = this.handleSearchChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.getReviews();
+  }
+
+  getReviews() {
+    axios.get('http://localhost:8000/reviews/4894941353195')
```

Comment:
this is a question we will need to figure out probably when we setup the proxy server. How do we pass the right isbn so that all services are aligned? As of now I did the same and hardcoded an isbn. 
Maybe instead of hardcoding it in the url, you can pass it as a parameter so that then it will be easier to make the change once we move to proxy.

Reply:
This is definitely something we still need to figure out.